sys/posix/inet/Kconfig

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_POSIX_INET
+    bool "Posix INet"
+    depends on MODULE_IPV4_ADDR
+    depends on MODULE_IPV6_ADDR
```

Comment:
For what I see in the Posix Inet module, it does not really depend on ipv4 and ipv6. It just adapts its behaviour when they are present. So it seems we can drop these dependencies.

Reply:
:cry: OK